app/src/main/java/com/github/se/wanderpals/MainActivity.kt

```diff
@@ -99,7 +100,21 @@ class MainActivity : ComponentActivity() {
             }
       }
 
-  private lateinit var placesClient: PlacesClient
+  private val locationPermissionRequest =
```

Comment:
Should the locationPermissionRequest be placed in the MainActivity file?

Reply:
Unfortunately yes since it uses a registerForActivityResult and the only Activity that we have is the MainActivity